back/engines/commercial/insights/app/controllers/insights/web_api/v1/inputs_controller.rb

```diff
@@ -48,8 +48,7 @@ def input
       def serialize_options()
         {
           include: %i[categories suggested_categories source],
-          fields: { idea: [:title_multiloc, :body_multiloc] },
-          params: { view: view }
+          params: { view: view, current_user: current_user }
```

Comment:
I think this is because I mistakenly replaced the fastjson_params instead of extending them to pass the view, got them back now, is it what you expected ?

Reply:
Yes, looks good :)